experimenter/experimenter/nimbus_ui_new/filtersets.py

```diff
@@ -302,8 +303,12 @@ def filter_type(self, queryset, name, value):
 
     def filter_takeaways(self, queryset, name, value):
         query = Q()
-        if NimbusExperiment.Takeaways.DAU_GAIN in value:
-            query |= Q(takeaways_metric_gain=True)
-        if NimbusExperiment.Takeaways.QBR_LEARNING in value:
-            query |= Q(takeaways_qbr_learning=True)
+        for val in value:
```

Comment:
Rename `value` to `values` and do `for value in values`

Reply:
so in all the methods like this for filtering we are using `def filter_takeaways(self, queryset, name, value):` as parameter name, I really don't want to change all the method name params to `values`, so that's why I did `val in value`

Reply:
I know I was thinking the same thing.  Both things are true.  But the argument is positional so we should be able to rename it to `values` without breaking the call interface.

Reply:
yeah then let me change other filters too from `value` to `values`